generic/tests/cfg/unattended_install.cfg

```diff
@@ -37,6 +37,16 @@
     # Fedora guests. For other guests like ubuntu if your install failed with
     # don't have enough RAM error from anaconda, please enlarge this value.
     lowest_mem = 512
+    variants:
+        - @default:
+        - multivm:
+            unattended_install_multivm = "yes"
+            thread_timeout = "3600"
+            variants:
+                - orderly:
+                    unattended_install_type = "orderly"
```

Comment:
Hi @balamuruhans would you like to explain what is the test purpose of the 'orderly' variant? IMHO, it looks the same as running this test multiple times orderly.

Reply:
@luckyh, Thank you for reviewing.
These scenarios are to closely simulate cloud environment and for scalability testing.
Example, if the cloud admin tries to launch 100-150 VMs,
1.  `orderly` testcase would spin each VM one by one (time between launch would be less than running multiple times)
2. `simultaneous` would spin VMs in parallel and random. 

This would exercise max limit, the way it tries to allocate memory from host, stress the virtualization stack where as running this test multiple times will also be another scenario but the time between launch each VM would be comparatively more.

Reply:
I can understand the test purpose of the `simultaneous` variant, but not the `orderly` variant. Because every single installation is executed one by one, they should be independent to each other, so I don't think it's reasonable to sum them up.

Reply:
one more difference is when we run the test multiple times for each VM, the VM will go to shutoff state when the test completes, so first VM will be in shutoff state when second VM is in installation/import. In this test the VMs are brought up one by one continuously and all the VMs will be up same time.

more over we don't have to run the test multiple times, let's achieve it in single test.